contracts/test/SequencerInbox.t.sol

```diff
@@ -19,24 +19,24 @@
 pragma solidity ^0.8.13;
 
 import "forge-std/Test.sol";
-import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
 import "../src/ISequencerInbox.sol";
 import "../src/libraries/Errors.sol";
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 import {SequencerInbox} from "../src/SequencerInbox.sol";
 import {Utils} from "./utils/Utils.sol";
 import {RLPEncodedTransactionsUtil} from "./utils/RLPEncodedTransactions.sol";
 
 contract SequencerBaseSetup is Test, RLPEncodedTransactionsUtil {
-    address internal sequencer;
     address internal alice;
     address internal bob;
+    address internal sequencerOwner;
+    address internal sequencerAddress;
 
     function setUp() public virtual {
-        sequencer = makeAddr("sequencer");
-
         alice = makeAddr("alice");
-
         bob = makeAddr("bob");
```

Comment:
If these accounts are not used pls delete them

Reply:
@saxenism can you do this and then we can merge

Reply:
Done. Please review again.